src/EnergyPlus/StandardRatings.cc

```diff
@@ -4691,7 +4691,7 @@ namespace StandardRatings {
                         std::tie(q, e, NetTotCoolCapBinned_2023, TotCoolElecPowerBinned_2023) =
                             IntermediateCapacityAndPowerSEER2Case2A(p_int, q_int, q_low, bl, n, Q_E_Int(1), q_full, P_E_Int(1), p_full, p_low);
                         goto SpeedLoop3_exit;
-                    } else if (bl < q_full) {
+                    } else { // bl < q_full
```

Comment:
[src/EnergyPlus/StandardRatings.cc:4694]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.